andela/wg-fortem
# 166152329 Update muscles list and exercises

#### Title
New muscles should be added to the existing muscle list and exercises updated to accommodate the new muscles.

#### Description
When a user navigates to the muscles overview page, they should be able to view the updated muscle list and corresponding exercises. The new muscles are; Lateral Deltoid, Obliquus Externus Abdominis, Posterior Deltoid, and Forearms.

#### Type of change








#### How Can This Be Tested?
To be able to test the feature, check out the branch `story/166152329-update-muscle-list`, run the command  `Invoke load-fixtures --settings-path path-to-your-settings-file` and then `python manage.py runserver`.
Navigate to the URL `http://127.0.0.1:8000/en/exercise/muscle/overview/` and you should be able to see the new muscles

Note that the Lateral Deltoid muscle is not easy to see but it is along the shoulder between the Anterior and Posterior Deltoid muscles

<img width="1440" alt="Screenshot 2019-06-11 at 20 18 35" src="https://user-images.githubusercontent.com/40719885/59292636-21acf480-8c86-11e9-9493-a0b26b0f1be7.png">


#### Checklist:
- [x] Add SVG files for new muscles
- [x] Update muscles fixtures with new muscles
- [x] Update exercises with the new muscles

#### PT stories
[#166152329](https://www.pivotaltracker.com/story/show/166152329)
